src/test/java/seedu/address/logic/parser/TaTrackerParserTest.java

```diff
@@ -16,22 +16,21 @@
 import seedu.address.logic.commands.AddCommand;
 import seedu.address.logic.commands.ClearCommand;
 import seedu.address.logic.commands.DeleteCommand;
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.EditCommand.EditPersonDescriptor;
 import seedu.address.logic.commands.ExitCommand;
 import seedu.address.logic.commands.FindCommand;
 import seedu.address.logic.commands.HelpCommand;
 import seedu.address.logic.commands.ListCommand;
```

Comment:
instead of this long list of imports, why not use `import seedu.address.logic.commands.CommandWords
`